tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/TaskProblemAction.java

```diff
@@ -59,9 +60,9 @@ public void run( final TaskProblem taskProblem, final ISelectionProvider provide
     {
         final IFile file = MigrationUtil.getIFileFromTaskProblem( taskProblem );
 
-        new Job( "Marking migration problem as done" )
+        new UIJob( "Marking migration problem as done" )
```

Comment:
I tried to use normal job to execute this. It  throw invalid thread exception, so i have to change it to UIJob, then it can work weel